tests/kafkatest/services/console_consumer.py

```diff
@@ -119,6 +123,22 @@ def __init__(self, context, num_nodes, kafka, topic, message_validator=None, fro
         self.message_validator = message_validator
         self.messages_consumed = {idx: [] for idx in range(1, num_nodes + 1)}
 
+        # Process client configuration
```

Comment:
You can simply do
`self.prop_file = self.render('console_consumer.properties')`

The properties file already does the "null/None" check:
```
{% if consumer_timeout_ms is defined and consumer_timeout_ms is not none %}
consumer.timeout.ms={{ consumer_timeout_ms }}
{% endif %}
```